aviary/interface/methods_for_level1.py

```diff
@@ -141,16 +140,6 @@ def run_level_1(
 
     prob = run_aviary(input_deck, phase_info, **kwargs)
 
-    if n2:
-        outfile = os.path.join(outdir, "n2.html")
-        if outdir != '':
-            os.makedirs(outdir, exist_ok=True)
-        om.n2(
-            prob,
-            outfile=outfile,
-            show_browser=False,
-        )
-
```

Comment:
why is this being removed? It does create the n2 file, as expected

Reply:
I removed it because `n2` is generated anyway no matter you have `--n2` option or not.

Reply:
After dicussion, I put it back, but still removed `--n2` flag.